sdk/metric/controller/basic/controller.go

```diff
@@ -131,6 +160,12 @@ func (c *Controller) Resource() *resource.Resource {
 	return c.resource
 }
 
+// Reader returns an InstrumentationLibraryReader for iterating
+// through the metrics of each registered library, one at a time.
+func (c *Controller) Reader() export.InstrumentationLibraryReader {
```

Comment:
```suggestion
// InstrumentationLibraryReader returns an InstrumentationLibraryReader for iterating
// through the metrics of each registered library, one at a time.
func (c *Controller) InstrumentationLibraryReader() export.InstrumentationLibraryReader {
```

Reply:
Yes, I should've done this in the first place!